TABTestKit/Classes/Biometrics/Biometrics.m

```diff
@@ -0,0 +1,39 @@
+//
+//  Biometrics.m
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 31/10/2018.
+//  Copyright © 2018 kane.codes. All rights reserved.
+//
+
+#import "Biometrics.h"
+#import "notify.h"
+
+@implementation Biometrics
+
++ (void)enrolled {
+    [self postEnrollment:true];
+}
+
++ (void)unenrolled {
+    [self postEnrollment:false];
+}
+
++ (void)successfulAuthentication {
+    notify_post("com.apple.BiometricKit_Sim.fingerTouch.match");
+    notify_post("com.apple.BiometricKit_Sim.pearl.match");
```

Comment:
Might be worth us making a note here that `pearl` is for Face ID and `fingerTouch` is Touch ID.